src/buildercore/terraform.py

```diff
@@ -645,14 +882,15 @@ def generate_delta(new_context):
     # simplification: unless Fastly is involved, the TerraformDelta will be empty
     # this should eventually be removed, for example after test_buildercore_cfngen tests have been ported to test_buildercore_cloudformation
     # TODO: what if the new context doesn't have fastly, but it was there before?
-    if not new_context['fastly'] and not new_context['gcs'] and not new_context['bigquery']:
+    managed_services = [k for k in MANAGED_SERVICES if new_context[k]]
```

Comment:
minor: maybe this could be named something like `used_managed_services` to differentiate it from the constant `MANAGED_SERVICES`.